libs/dh/admin/feature-user-management/src/lib/tabs/users-tab/dh-users-tab.component.spec.ts

```diff
@@ -93,11 +93,13 @@ describe(DhUsersTabComponent, () => {
     };
   }
 
-  it('displays user data', async () => {
+  it('displays user data', fakeAsync(async () => {
     await setup(users);
 
     const [testUser] = users;
 
+    tick(250);
```

Comment:
suggestion: let's move this value to a variable in the component and import it from there. Otherwise the tests will start to fail if the value is changed in the component but not here.

Reply:
Good idea